### ISSUE TITLE
[Feature Request] Move to local manga

### ISSUE THREAD

**Issue by pervertkirito** [inorichi/tachiyomi#2289]
After I update extension today, I noticed manga  source that I use got [removed](https://github.com/inorichi/tachiyomi-extensions/pull/1653).
why not add feature where you can move downloaded manga from removed source into local manga

**Comment by joseph619** [comment]
This is not app issue, post this on Tachiyomi extension repo

**Comment by arkon** [comment]
Ignore that, this isn't an extension issue ^

**Comment by joseph619** [comment]
I removed my comment, not sure if this could be properly implemented since each source have many manga with different chapter count from another source (some source upload ch 0 as ch 1)

**Comment by happywillow0** [comment]
All we need to do is check for orphaned downloads and move them over. 

We can also optionally filter it to just manga with a empty chapter list indicating that the source removed the manga completely.

The coding is always the harder part ;)

**Comment by CarlosEsco** [comment]
Just cause a manga source is removed doesn't mean the manga gets removed from the library

**Comment by happywillow0** [comment]
But if you do a refresh, wouldn't the chapter get cleared? It happen with the mangadex mangas.

**Comment by CarlosEsco** [comment]
i dont think it will. because there source is completey gone at that point.  It should just error out. THe mangadex one the manga url exists.  The chapters got removed

**Comment by happywillow0** [comment]
Yea, your right. Different scenarios and different logic paths. 
It would be nice to be able to identify and move / delete downloaded chapters that get orphaned. 

Actually in the case of the original request, now that I actually followed the link, since the source is gone, the library entry should still be there (as you mentioned) and the downloaded manga should load. 

Otherwise, the simplest workaround would be to use a file manager and move the downloads into the local folder manually.

**Comment by joseph619** [comment]
Yes, moving all downloads to local should work.
